.github/workflows/tests.yml

```diff
@@ -262,7 +263,7 @@ jobs:
 
       - name: Ember Build for System Tests
         if: matrix.build_type == 'system'
-        run: bin/ember-cli --build
```

Comment:
Doing this so that the new script is exercised *somewhere* in GitHub CI.

The 'incremental' side of it will also be exercised in our internal smoke test.

Testing production builds in more detail is quite expensive, so avoiding it for now